### ISSUE TITLE
package ‘rsm’ is not available (for R version 3.2.2) 

### ISSUE THREAD

**Issue by tobigithub** [rietho/IPO#17]
Hi,
package ‘rsm’ is not available (for R version 3.2.2)

this is under WIN7 with R3.2.2 (64bit). Basically after installing a number of
R packages from 3.2.15 to 3.2.0 and other is basically either this or devtools
throwing errors. One solution is 

> setRepositories()
> install.packages("rsm")

But one could also install and compile from source
https://cran.r-project.org/web/packages/rsm/index.html

Quite interesting updating R and packages always breaks
multiple things, maybe some Docker type R solutions would be good,
basically to freeze packages and installs.
https://www.google.com/search?num=20&q=docker+for+R

Cheers
T.

**Comment by glibiseller** [comment]
Hi,

I just installed R3.2.2 (32bit) on my WIN7 machine. I had no problems installing rsm choosing an austrian server. Does is still not work for your setup?

Cheers, 
Gunnar
